ml_model/src/main/java/edu/wm/sealab/featureextraction/Parser.java

```diff
@@ -91,11 +101,19 @@ public static void main(String[] args) {
                     new SyntacticFeatureExtractor(featureVisitor.getFeatures());
                 Features features = sfe.extract(cuNoComm.toString());
```

Comment:
we should not parse the complete file to extract features. - We need to parse only the method.
Refer https://github.com/sea-lab-wm/complexity-verification-project/pull/11/files#diff-7abf8e148e543419c5e269a9deac66738c0e2ca2b23656b27ddbabddbf376f27